packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -43,10 +43,13 @@ typedef void GestureDragCancelCallback();
 ///
 /// See also:
 ///
-///  * [HorizontalDragGestureRecognizer]
-///  * [VerticalDragGestureRecognizer]
-///  * [PanGestureRecognizer]
+///  * [HorizontalDragGestureRecognizer], for left and right drags.
+///  * [VerticalDragGestureRecognizer], for up and down drags.
+///  * [PanGestureRecognizer], for drags that are not locked to a single axis.
```

Comment:
It's not so much that they're in a particular direction with Pan, it's that they literally can go in multiple directions.